tests/functional/test_data_classes.py

```diff
@@ -1201,6 +1202,30 @@ def test_kafka_self_managed_event():
     assert record.get_header_value("HeaderKey", case_sensitive=False) == b"headerValue"
 
 
+def test_kinesis_firehose_event():
```

Comment:
Could you please add another test with a different payload? You can get the payload examples here:

1 - [KinesisStream](https://github.com/awslabs/aws-lambda-powertools-python/pull/1556/files#diff-b06a860206a2cce04772240219617495997d9ccd3faadaf4488b160e9abda0f1)  as source
2 - [PUT](https://github.com/awslabs/aws-lambda-powertools-python/pull/1556/files#diff-0d2c331ab6fe3cb1e3e7c28b0918a588bfeca1eacad31b166a5bf6f4599b9f02) as source

Reply:
done